Submitted By: Ken Moffat <ken at linuxfromscratch.org>
Date: 2005-11-29
Initial Package Version: 1.3.13
Upstream Status: Submitted and merged
Origin: fedora
Description: Fixes ofpath (needed for initial install) for SATA disks,
following the flattening of /proc/device-tree circa 2.6.12.

--- yaboot-1.3.12/ybin/ofpath.nobootx	2004-04-17 20:22:00.000000000 -0400
+++ yaboot-1.3.12/ybin/ofpath	2005-08-04 07:47:19.000000000 -0400
@@ -425,14 +425,6 @@
 {
     case "$DEVNODE" in
 	sd*)
-	    if ls -l /proc/device-tree | grep -q ^lr ; then
-		true
-	    else
-		echo 1>&2 "$PRG: /proc/device-tree is broken.  Do not use BootX to boot, use yaboot."
-		echo 1>&2 "$PRG: The yaboot HOWTO can be found here: http://www.alaska.net/~erbenson/doc"
-		return 1
-	    fi
-
 	    ## use common scsiinfo function to get info we need.
 	    scsiinfo || return 1
 
